### PR TITLE
hoc2015: a couple fixes to puzzle 14.

### PR DESCRIPTION
**Pull request by breville**
- Success criteria now checks for the two settings we want (droid & droid speed)
- Simplified success criteria
- Blockly version toolbox now has default "fast" speed in setDroidSpeed block to match droplet version.

### CODE REVIEW

**Comment by breville** [comment]
fyi @islemaster and @philbogle

**Comment by philbogle** [comment]
No concerns here, very targeted.

File: apps/src/studio/levels.js [hunk 1]
```diff
@@ -2977,16 +2977,14 @@ levels.js_hoc2015_change_setting = {
   ],
   'progressConditions' : [
     // Collected all the items and set the right properties?  Success.
-    { required: { 'setMap': true, 'setDroidSpeed': true, 'collectedItemsAtOrAbove': 3 },
+    { required: { 'setDroid': true, 'setDroidSpeed': true, 'collectedItemsAtOrAbove': 3 },
```
Comment (cpirich) [inline apps/src/studio/levels.js]: Is there a `setDroid` option for `progressConditions`? I don't see it in `Studio.conditionSatisfied`

There is a `hasSetDroid` which is initialized to `false`, but never set.
There is a `hasSetSprite` which is never initialized/reset, but is set in `callCmd`

The droplet levels do call a `setDroid` api, which calls `queueCmd` as `setSprite`
The blockly levels always generate `setSprite` directly, bypassing the `setDroid` API.

**Comment by cpirich** [comment]
I think you're going to need to make additional code changes if you really want to require `setDroid` - it would be easier to change the instructions to indicate that you need to set the map :smile:

File: apps/src/studio/levels.js (hunk 1 above)
Comment (breville) [inline apps/src/studio/levels.js]: doh, great catch.  Should be fixed now, and tested in both blocks & non-blocks.  Thanks!

**Comment by cpirich** [comment]
:+1:

**Pull request merged**